stratum/hal/lib/phal/onlp/onlp_wrapper.cc

```diff
@@ -127,6 +128,9 @@ ::util::Status OnlpWrapper::Initialize() {
   LOAD_SYMBOL(onlp_led_mode_set);
   LOAD_SYMBOL(onlp_led_char_set);
   LOAD_SYMBOL(onlp_psu_info_get);
+  LOAD_SYMBOL(onlp_i2c_mux_select);
```

Comment:
LOAD_SYMBOL_IF_PRESENT(onlp_sfp_frequency_set);
LOAD_SYMBOL_IF_PRESENT(onlp_sfp_mux?_get);

Nullptr if not present, then check before using the function